.claude/agents/review-worklist-writer.md

```
@@ -0,0 +1,69 @@
+   - Where (file:line)
+   - How (concrete suggestion when available)
+
+5. **Identify follow-up work**
+   - Items that should be separate PRs
+   - Nice-to-have improvements
+
+## Status Reporting
+
+- VERIFIED: Worklist created with prioritized items
+- UNVERIFIED: Worklist created but classification uncertain
+- BLOCKED: No feedback to process
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)